### ISSUE TITLE
Feature Request: Add menu in GUI to access pfm_documentation_url, pfm_app_url links, and the payload manifest

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileCreator#135]
The thought occurred to me today that for folks that want to contribute to an existing manifest, getting to the related documentation from within the GUI would be helpful, rather than having to navigate and open the manifest itself.  This feature would simply incorporate the links that are already provided in the following keys:
`pfm_app_url`
`pfm_documentation_url`

Because there is the opportunity to supply both manifest-level `pfm_app_url` and `pfm_documentation_url` keys as well as for individual preferences (Google Chrome is a great example), I envision a couple possibilities:

1. Have a physical button - an arrow, three dots, or something else that makes sense - adjacent to the individual preference or at the top by the payload title which would reveal a menu.

2. When right-clicking on a preference key name or the payload name, reveal this menu.

The menu would be similar to what's produced when right-clicking on something, and list "Open app url" and "Open documentation url" options, which would then open in the default web browser.

Along the same lines, having a link or menu item that could take you directly to a payload's manifest in the Finder would be helpful as well.

**Comment by erikberglund** [comment]
Yes. This will have to be added. I am first going to add the os version the key supports as a footer and see how that looks. That space could be used by more things aswell.

and absolutely a context menu.

In my Objective-C prototype of this app I had another view on the left side with all of this info where you could show it by clicking a payload or in the toolbar, and that could show all different info about a payload. But that is also a lot of screen space with much whitespace as some keys don't have that much to show.

But this will have to be added, absolutely.

**Comment by apizz** [comment]
Documentation links added and working in latest beta build

**Comment by apizz** [comment]
@erikberglund individual pref documentation links are added, but not payload-level documentation & app links

**Comment by erikberglund** [comment]
Yes, I will put that up for the version after this.
